ipytest/cov.py

```diff
@@ -0,0 +1,106 @@
+"""A coverage.py plugin to support coverage in Jupyter notebooks
+
+The plugin must be enabled in a `.coveragerc` next to the current notebook or
+the `pyproject.toml` file. See the [coverage.py docs][coverage-py-config-docs]
+for details. In case of a `.coveragerc` file, the minimal configuration reads:
+
+```ini
+[run]
+plugins =
+    ipytest.cov
+```
+
```

Comment:
The `%%file` cell magic can be used to create files like `.coveragerc` from a notebook.
```
%%file .coveragerc
[run]
plugins =
    ipytest.cov
```

Would an `ipytest.generate_coveragerc()` or similar be too much? or Just `autoconfig(coverage=True)` or something?

Reply:
TBH. I would rather prefer, that there is an option to configure pytest-cov programatically and inject the plugin in that way. I will open an issue for further discussion.